zooniverse/front-end-monorepo
# Refactor classifier layout per survey task

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- closes #6237 , closes #5093 

## Describe your changes
- add `usesSurveyTask` view to Workflow store
- pass `usesSurveyTask` from Layout to CurrentLayout
- update CenteredLayout, MaxWidth, and NoMaxWidth with larger task area if `usesSurveyTask`
- MaxWidth and NoMaxWidth between 769px-1120px use 9fr 5fr for viewer task areas, these changes include aligning viewer and task vertically at 1119px if `usesSurveyTask`
- center task nav buttons (Done & Talk, Done, Next, Back)

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - staging test project with survey task: https://local.zooniverse.org:3000/projects/markb-panoptes/survey-task-test-project/classify/workflow/2434?env=staging
- What user actions should my reviewer step through to review this PR?
  1. load workflow with survey task
  2. note wider task area with survey task
  3. change display width and note how layout changes (horizontal vs vertical viewer and task alignment)
- Which storybook stories should be reviewed?
  - http://localhost:6006/?path=/story/layouts-centered--with-survey-task
  - http://localhost:6006/?path=/story/layouts-maximum-width--with-survey-task
  - http://localhost:6006/?path=/story/layouts-no-max-width--with-survey-task
- Are there plans for follow up PR’s to further fix this bug or develop this feature? see other survey task issues

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
